### ISSUE TITLE
GZip not working for JavaScript (incl. proposed fix)

### ISSUE THREAD

**Issue by infinitnet** [serghey-rodin/vesta#412]
First of all thanks for your hard work - Vesta is one of my favourite control panels. However I noticed an issue with a missing MIME type in your nginx.conf, which causes some JavaScript not to be gzipped correctly.

Current gzip config in nginx.conf:
```
    gzip_types          text/plain text/css text/javascript
                        application/x-javascript;
```

Proposed fix that assures all JS gets compressed:
```
    gzip_types          text/plain text/css text/javascript
                        application/x-javascript application/javascript;
```

Without the `application/javascript` MIME type added, the JS of several CMSes doesn't get compressed  (tested with Joomla! for example).

**Comment by infinitnet** [comment]
Is anyone here caring of the pull requests? There are so many pending ones and I'd like to know if it's worth it spending time on improving/fixing Vesta or if the pull requests never get merged anyway?

**Comment by infinitnet** [comment]
Fix merged to master via #421
